bitnami/redis/templates/configmap.yaml

```diff
@@ -42,7 +42,7 @@ data:
     dir "/tmp"
     bind 0.0.0.0
     port {{ .Values.sentinel.port }}
-    sentinel monitor {{ .Values.sentinel.masterSet }} {{ template "redis.fullname" . }}-master-0.{{ template "redis.fullname" . }}-headless.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }} {{ .Values.redisPort }} {{ .Values.sentinel.quorum }}
+    sentinel monitor {{ .Values.sentinel.masterSet }} {{ template "redis.fullname" . }}-node-0.{{ template "redis.fullname" . }}-headless.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }} {{ .Values.redisPort }} {{ .Values.sentinel.quorum }}
```

Comment:
This chart still uses the "master/slave" terminology, see:
- https://github.com/bitnami/charts/blob/master/bitnami/redis/templates/redis-master-statefulset.yaml#L5
- https://github.com/bitnami/charts/blob/master/bitnami/redis/templates/redis-master-statefulset.yaml#L18
Therefore, the hostnames associated to these nodes will include `-master` instead of `node`. 

Please note this hostnames are generated based on the names we give to the Redis stafulsets & pods and are not related with Redis terminology.